docker/.dockerignore

```diff
@@ -0,0 +1,10 @@
+.git
+.gitignore
+node_modules
+npm-debug.log
+Dockerfile*
+docker-compose*
+README.md
+LICENSE
+.vscode
+package-lock.json
```

Comment:
@alex-gagnon Do we really need to ignore this file? I don't see anything sensitive in here. If we ignore this and folks need it to get stuff working and there is nothing sensitive, then this should be included in the repo when people pull down and install.

Reply:
.dockerignore just ignores it from creating the Docker image/container. It doesn't prevent it from being in the repo.

Reply:
So my thinking is not about .dockerignore preventing the files inside .dockerignore from being included in the repo but about ".dockerignore" being added to ".gitignore" so that users don't have ".dockerignore" to tell docker what to ignore when they pull down the repo.

For example, if we have .env in .gitignore, people would have to create their own .env file which is ok because .env would contain secrets. If .dockerignore is ignored, then people would have to recreate it on their end which doesn't make sense to add that extra step when everyone would have the same .dockerignore file and there are no secrets in there.

Reply:
Or is this not being added to .gitignore because it's not at the root level of the repo?